### PR TITLE
Sidebar v2

### PR DESCRIPTION
**Pull request by hawkticehurst**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -655,9 +652,32 @@ button, input {
...
+}
+
+.sidebar-header span {
+  cursor: pointer;
+}
+
 .sub-module-progress-row-container {
-  border-bottom: 1px solid white;
-  padding: 5px;
+  /* border-bottom: 1px solid white; */
```
Comment (ConnerArdman) [inline src/App.css]: We can delete this line

File: src/components/learning-module/SideBar.js [hunk 2]
```diff
@@ -45,9 +45,12 @@ function SideBar(props) {
           className={'sidebar ' + animationClass}
           onClick={stopPropagation}
           onAnimationEnd={onAnimationEnd}>
-          <div className="go-back-summary" onClick={onClickGoBackSummary}>
-            <img src={require('./images/arrow-left.svg')} alt="Arrow Left Icon" />
-            <p>{'Back To Summary'}</p>
+          <div className="sidebar-controls">
+            <div className="go-back-summary" onClick={onClickGoBackSummary}>
+              <img src={require('./images/arrow-left.svg')} alt="Arrow Left Icon" />
+              <p>{'Back To Summary'}</p>
```
Comment (ConnerArdman) [inline src/components/learning-module/SideBar.js]: No need for {'text'} syntax for strings. Its just HTML syntax:
```<p>Back To Summary</p>```

Comment (hawkticehurst) [reply src/components/learning-module/SideBar.js]: Lol that was actually something that whoever created this component used ;)

Comment (hawkticehurst) [reply src/components/learning-module/SideBar.js]: Changing now however.

**Pull request merged**